src/unix/core.c

```diff
@@ -965,11 +965,10 @@ int uv__fd_exists(uv_loop_t* loop, int fd) {
   return (unsigned) fd < loop->nwatchers && loop->watchers[fd] != NULL;
 }
 
-
-int uv_getrusage(uv_rusage_t* rusage) {
+int uv__getrusage(int who, uv_rusage_t* rusage) {
```

Comment:
```suggestion

int uv__getrusage(int who, uv_rusage_t* rusage) {
```
Style nit: Should we keep a two line gap between these functions like most functions here?